repo2docker/buildpacks/r.py

```diff
@@ -141,6 +133,19 @@ def detect(self):
                 self._runtime = "r-{}".format(str(self._checkpoint_date))
             return True
 
+    def get_env(self):
+        """
+        Set custom env vars needed for RStudio to load
+        """
+        return super().get_env() + [
+            # rstudio (rsession) can't seem to find R unless we explicitly tell it where
+            # it is - just $PATH isn't enough. I discovered these are the env vars it
+            # looks for by digging through RStudio source and finding https://github.com/rstudio/rstudio/blob/main/src/cpp/r/session/RDiscovery.cpp
```

Comment:
```suggestion
            # looks for by digging through RStudio source and finding https://github.com/rstudio/rstudio/blob/54f1090ab32d9670f075a76bd575a8e0e2c26e25/src/cpp/r/session/RDiscovery.cpp#L57-L74
```

If this changes, for example the code logic is moved, its helpful to know how the logic looked historically so you can search for its removal or relocation. So, I suggest adjusting this link to a permalink.

Reply:
@consideRatio github seems to have fucked this up somehow, the latest code already points to a git tag here.